packages/@sanity/base/src/components/previews/general/DetailPreview.styled.tsx

```diff
@@ -0,0 +1,32 @@
+import {Box, Flex, rem, Stack} from '@sanity/ui'
+import styled from 'styled-components'
+import {PREVIEW_MEDIA_SIZE} from '../constants'
+
+export const Root = styled(Flex)`
+  user-select: none;
+  height: ${rem(PREVIEW_MEDIA_SIZE.detail.height)};
+  box-sizing: border-box;
```

Comment:
This can be replaced with `sizing="border"` on the `Root` component. Perhaps it should be called `RootFlex` to follow the same naming as other components here!